.github/workflows/build-release-doc-pages.yml

```diff
@@ -0,0 +1,32 @@
+name: Generate github pages
+
+on:
+  push: 
+    branches: [ feature/documentation-pages, main ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          ref: feature/documentation-pages
+
+      - name: Install asciidoctor
```

Comment:
Could you provide a bash script or python script then in local, anyone can run and verify?